### ISSUE TITLE
Item table & Other Details are overlapping

### ISSUE THREAD

**Issue by dineshpanchal93** [frappe/print_designer#103]
In Item table we have almost 14 items
In Words & other details overlapping with each other

https://github.com/frappe/print_designer/assets/21578167/f2dd12f1-1ee1-4ac5-be84-a89d60ad295c

**Comment by maharshivpatel** [comment]
@dineshpanchal93 yes this is known issue and actual fix for this is #102 but this is deal breaker for many people like you i will add workaround in 1-2 days. i will inform you once it is done. Thank you for sharing the issue.

**Comment by maharshivpatel** [comment]
I have already done the workaround but i am not pushing it as i am attempting #102 . If i am unable to do it soon i will push the workaround. 

reason for not pushing workaround is it changes some internal structure that i would like to avoid.

**Comment by dineshpanchal93** [comment]
@maharshivpatel Thank for the update.

You can take your time you can create the diff branch so we can also test the same.

**Comment by maharshivpatel** [comment]
@dineshpanchal93 can you please check if it is working fine now. 

you can check pr to see how to setup basically you set items table as primary from properties panel

**Comment by dineshpanchal93** [comment]
ok Thanks, Will update

**Comment by dineshpanchal93** [comment]
Yes, its working fine now.

but still, there is a catch for the 2nd page.

**Comment by maharshivpatel** [comment]
That will be solved by #102. i am brainstorming how to do it.

**Comment by dineshpanchal93** [comment]
@maharshivpatel ok thanks 

Thanks for the efforts !!
